### DISCUSSION TITLE
add maps and mods

### DISCUSSION THREAD

**Discussion by BattleHook** [Zweistein2/AgrarMonitor#3] [Ideas]
how can I help you with maps and mods to incorporate ?

**Comment by Zweistein2** [comment]
To add maps I need:
- A high resolution picture of the map
- All data for fields (position and size)
- All data for new crops (name, mass/l, price/l, priceFactor per Month, yield/m², seedUsage/m², chaffFactor, growCycles, if it produces straw and if it is boosted by bees)

For mods I need:
- Translations
- All data for new vehicles (filename, type, brand, category, name)
- All data for new placeables (filename, name)
- All data for new crops (name, mass/l, price/l, priceFactor per Month, yield/m², seedUsage/m², chaffFactor, growCycles, if it produces straw and if it is boosted by bees)

But you can tell me which mods / maps to support. I will try to add them to the next release.

**Reply by BattleHook** [reply]
maybe you can make import from modfile like here : [FS19_Webstats](https://github.com/J0hnHawk/FS19_Webstats).

I had this running on a NAS for my dedicates server. worked like a charm.

**Reply by Zweistein2** [reply]
I will implement something like this for the next release

**Reply by BattleHook** [reply]
I really play Papenburg Map version 1.0.0.5 a lot. Maybe you can incorporate this ?

**Reply by Zweistein2** [reply]
Will be added in the next update
